test/data/repositories/fedora-38.json

```diff
@@ -0,0 +1,30 @@
+{
+  "x86_64": [
+    {
+      "name": "fedora",
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f37/f37-x86_64-fedora-development-20221025/",
```

Comment:
The file is `fedora-38.json`, but the URL contains `f37` which does not look right 🤔

Reply:
That's interesting because I *think* that file got copied when I generate manifests. Let me recheck.